static/js/containers/VideoDetailPage_test.js

```diff
@@ -150,6 +152,17 @@ describe('VideoDetailPage', () => {
     });
   });
 
+  it('show the embed dialog', async () => {
+    let wrapper = await renderPage({editable: true});
+    await listenForActions([
+      SET_SHARE_DIALOG_VISIBILITY,
+      CLEAR_SHARE_DIALOG,
+    ], () => {
+      wrapper.find(".share").props().onClick();
+      wrapper.find("Dialog").at(1).props().onCancel();
+    });
+  });
```

Comment:
it might be good to add another test just checking that the embed URL and share URL are rendered correctly

Reply:
OK